.github/workflows/docs.yml

```
@@ -0,0 +1,57 @@
+name: Docs
+on:
+  pull_request:
+    types: [synchronize, edited, opened]
+  push:
+    branches: [main]
```

Er... I may be wrong about how this works, but if this is supposed to be a string match for the branch name, as opposed to `main` being magical:
```suggestion
    branches: [master]
```

(Same in `spellcheck.yml`. And, no, I do not intend to rename the primary branch.)